### ISSUE TITLE
Peer review virtual pollutant loading calcs

### ISSUE THREAD

**Issue by austinorr** [Geosyntec/StormPiper#158]
## Peer review of loading calculations.

Calculate the load of DEHP, PYR and PHE for an example feature given below using ee to compute the runoff volume, tss load, tss conc, and then convert to our 3 virtual params. This will give us a  strong check on the calculation pipeline and unit conversions.

I think you've already got a notebook doing some/most of this, just needs a revisit. 

The application currently uses these ee layers for the calculations, please do the calcs with these same layers:

- EE_RUNOFF_PATH = "projects/ee-stormwaterheatmap/assets/production/Mean_Annual_Q_4_epochs"
- EE_COC_PATH = "projects/ee-tacoma-watershed/assets/production/coc_concentrations"

Please use this feature from the internal [examples](https://github.com/Geosyntec/StormPiper/blob/273b42d37bfd6eccfcec75051ed636c6abf7b0ee/stormpiper/stormpiper/models/scenario.py#L56):
```
{
    "type": "FeatureCollection",
    "features": [
        {
            "type": "Feature",
            "properties": {"name": "delin-01"},
            "geometry": {
                "type": "Polygon",
                "coordinates": [
                    [
                        [-122.473, 47.250],
                        [-122.466, 47.250],
                        [-122.466, 47.255],
                        [-122.473, 47.255],
                        [-122.473, 47.250],
                    ]
                ],
            },
        }
    ],
}
```

For completeness and traceability please report the following in the system's standard units so we can discover the cause of any variances between our methods: 

- runoff volume in units of cuft
- pollutant load in units of lbs [TSS, PYR, DEHP, PHE]
- concentrations in mg/l [TSS, PYR, DEHP, PHE]

**Comment by cnilsen** [comment]
Runoff Volume:
```
Runoff Volume 1980s, cuft: 3465424.2839802
```
Pollutant loads: 
@austinorr Here are my results. The script to calculate these is here: https://code.earthengine.google.com/0d7fad96f77a55510e6a4be5fbff3ceb

I'll save it to our repo - let me know where the best place is. 

```
TSS Load (lbs)
4992.681001120828

{
  "dehp": 0.22162510963975354,
  "phe": 0.004992681001120828,
  "pyr": 0.009486093902129575
}
```
Concentrations 
```
TSS Concentration (mg/L)
23.07498241078546
{
  "dehp": 0.0010242984692147665,
  "phe": 0.00002307498241078546,
  "pyr": 0.00004384246658049237
}
```

**Comment by austinorr** [comment]
@cnilsen, we're super close, but my values are slightly larger. There are a couple differences in our EE implementation related to scale (yours is set to 10, mine is set to 1) and area calc (you're using delin_m2, i'm using Image.pixelArea()). The net effect appears to be a bit more runoff and load is generated in my calc, but the values are really close -- same order of magnitude and really similar absolute value.

If you think the differences in implementation described above are likely to boost my values relative to yours, then i think we can call this calculation *checked* and **passed**.

Thanks for your help, here are my values:

```
"delin_load": [
  {
  "altid": "edd2891d",
  "epoch": "1980s",
  "units": "lbs",
  "value": 0.24981570632300265,
  "variable": "DEHP"
  },
  {
  "altid": "edd2891d",
  "epoch": "1980s",
  "units": "lbs",
  "value": 0.005627747382811504,
  "variable": "PHE"
  },
  {
  "altid": "edd2891d",
  "epoch": "1980s",
  "units": "lbs",
  "value": 0.010692720027341856,
  "variable": "PYR"
  },
  {
  "altid": "edd2891d",
  "epoch": "1980s",
  "units": "lbs",
  "value": 3.429676500252002,
  "variable": "TCu"
  },
  {
  "altid": "edd2891d",
  "epoch": "1980s",
  "units": "lbs",
  "value": 250.68873241158374,
  "variable": "TN"
  },
  {
  "altid": "edd2891d",
  "epoch": "1980s",
  "units": "lbs",
  "value": 15.154725244605203,
  "variable": "TP"
  },
  {
  "altid": "edd2891d",
  "epoch": "1980s",
  "units": "lbs",
  "value": 5627.747382811504,
  "variable": "TSS"
  },
  {
  "altid": "edd2891d",
  "epoch": "1980s",
  "units": "lbs",
  "value": 23.11815485222645,
  "variable": "TZn"
  },
  {
  "altid": "edd2891d",
  "epoch": "1980s",
  "units": "cuft",
  "value": 3510460.99969192,
  "variable": "runoff"
  }
]
```

**Comment by austinorr** [comment]
@cnilsen I looked through the script again, and it looks like you're calculating the load as the avg area conc * the avg area runoff. This certainly simplifies the calculation, but also leads to strong undercounting of load for large areas since the avg conc of a large area will always be diluted by low polluting (and low runoff generating) areas. 

Instead, the calculation needs to consider both the runoff generated and the co-occurring concentration (per pixel or per 'scale' value) and compute the zonal stats with the `reduceRegions` on the volume load and the pollutant load. I think that using the 'mean' reducer function on the concentration raster is an error, and the calc should be done on a calculated load raster instead. Then for a given region we'd calculate concentration for the zone by load poc / volume at the end.

**Comment by cnilsen** [comment]
@austinorr 
I checked and you are correct. I updated the script and it is outputting reasonable values. 
Here are the load results in lbs/year: 

```
{
  "tss": 5630.120674786363
  "dehp": 0.24992105675376666,
  "phe": 0.005630120674786363,
  "pyr": 0.01069722928209409
}
```

Regarding the conversion factors, here are the factors for converting from TSS concentration to the surrogate parameters:

```
{
    'phe': 1e-6,
    'dehp': 4.439e-5,
    'pyr': 1.9e-6
}  
```
The conversion factors are unitless. So in my calcs, I started with the TSS raster which was in μg/L and used these factors to convert and keep in units of ug/L. 

For reporting _concentrations_ I think it makes sense to keep everything in units of μg/L except for TSS which should be in mg/L.
